### ISSUE TITLE
svtplay-dl 2.3 used on svtplay.se: issues

### ISSUE THREAD

**Issue by gustaf71** [spaam/svtplay-dl#1136]
### svtplay-dl versions:

svtplay-dl 2.3

### Operating system and Python version:

Python 3.7.4

### What is the issue:

svtplay-dl --verbose https://www.svtplay.se/video/23518238/the-sinner/the-sinner-sasong-2-avsnitt-1

DEBUG [1567797202.922694] /usr/local/lib/python3.7/site-packages/svtplay_dl/utils/getmedia.py/get_media: version: 2.3
DEBUG [1567797202.929555] /usr/local/lib/python3.7/site-packages/svtplay_dl/service/__init__.py/__init__: service: svtplay
DEBUG [1567797202.9297721] /usr/local/lib/python3.7/site-packages/svtplay_dl/utils/http.py/request: HTTP getting 'https://www.svtplay.se/video/23518238/the-sinner/the-sinner-sasong-2-avsnitt-1'
DEBUG [1567797202.9562721] /usr/local/lib/python3.7/site-packages/urllib3/connectionpool.py/_new_conn: Starting new HTTPS connection (1): www.svtplay.se:443
DEBUG [1567797203.60344] /usr/local/lib/python3.7/site-packages/urllib3/connectionpool.py/_make_request: https://www.svtplay.se:443 "GET /video/23518238/the-sinner/the-sinner-sasong-2-avsnitt-1 HTTP/1.1" 200 27911
Traceback (most recent call last):
  File "/usr/local/bin/svtplay-dl", line 7, in <module>
    svtplay_dl.main()
  File "/usr/local/lib/python3.7/site-packages/svtplay_dl/__init__.py", line 64, in main
    get_media(urls[0], config, __version__)
  File "/usr/local/lib/python3.7/site-packages/svtplay_dl/utils/getmedia.py", line 56, in get_media
    get_one_media(stream)
  File "/usr/local/lib/python3.7/site-packages/svtplay_dl/utils/getmedia.py", line 106, in get_one_media
    for i in streams:
  File "/usr/local/lib/python3.7/site-packages/svtplay_dl/service/svtplay.py", line 81, in get
    self.outputfilename(janson)
  File "/usr/local/lib/python3.7/site-packages/svtplay_dl/service/svtplay.py", line 254, in outputfilename
    season, episode = self.seasoninfo(data)
  File "/usr/local/lib/python3.7/site-packages/svtplay_dl/service/svtplay.py", line 286, in seasoninfo
    season = "{:02d}".format(match.group(1))
ValueError: Unknown format code 'd' for object of type 'str'

**Comment by spaam** [comment]
thanks for the bug repot <3. it should be fixed in master now.  will release 2.4 tomorrow with the fix.

**Comment by gustaf71** [comment]
That is great and thanks for your fast respond. 

I’m not a programmer but I really like to understand your code in python.

So I hope to see more comments in your code ;)

**Comment by gustaf71** [comment]
I will do more testing when 2.4 is ready and let you know.

**Comment by spaam** [comment]
i dont have so much comments sadly . i should add more :)

**Comment by gustaf71** [comment]
I will be happy if you do.

I have test 2.4 on the same content everything was working fine. 

The change you did made the config file to be incorrect.
Here is my example of config file:
service:
    svtplay:
        filename: '{title}.s{season}e{episode}-{service}.{ext}'

And here is the result I get with svtplay-dl version 2.4
the-sinner.s02e01.avsnitt-1-4a6fbba-svtplay.ts

So if I understand correct {episodename}-{id} are included and older version did not.
In other words the config file was working with older version.

So maybe there have been a 'error' somewhere in your code concern config file and the options.
